library/src/webapp/js/mathjax-config.js

```diff
@@ -0,0 +1,48 @@
+let scriptTag = document.createElement("script");
+scriptTag.type = "text/x-mathjax-config";
+
+let format = (portal.mathJaxConfig.format);
+let ext = [];
+let input = [];
+
+function useDefaultFormat(){
+    ext.push("tex2jax.js");
+    input.push("input/TeX");
+}
+
+if (format === undefined) {
+    console.error("No property for MathJax config was specified. Using LaTeX as default.");
+    useDefaultFormat();
+} else {
+    format.forEach(extension => {
+        switch (extension) {
+            case "LaTeX":
+                useDefaultFormat();
+                break;
+            case "AsciiMath":
+                ext.push("asciimath2jax.js");
+                input.push("input/AsciiMath");
+                break;
+            default:
+                console.error(extension + " is not a supported extension." +
+                " Check available options on Sakai default properties");
+                break;
+        }
+    });
+}
+
+if (ext.length === 0) {
+    console.error("None of the received formats match the supported ones. Using LaTeX as default.");
+    useDefaultFormat();
+}
+
+let confObject = {
+    extensions: ext,
+    jax: [...input, "output/HTML-CSS"],
+    messageStyle: "none",
+    tex2jax: { inlineMath: [['$$','$$'],['\\(','\\)']] }
```

Comment:
Could this part be configurable?
We have this inlineMath: [['$','$'], ['\\\\(','\\\\)']] }

Reply:
Of course, the idea is to be able to update the method to accept more properties in the future.

Maybe this is the most complex case. Would you offer predefined options, or let the end user add custom delimiters?
For the second case, could we easily add different symbols for the start and the ending through properties?
Also, how many delimiters should we accept?